lib/src/builder.dart

```diff
@@ -78,10 +78,13 @@ class GeneratorBuilder extends Builder {
       genPartContent = formatter.format(genPartContent);
     } catch (e, stack) {
       buildStep.logger.severe(
-          """Error formatting the generated source code.
-This may indicate an issue in the generated code or in the formatter.
-Please check the generated code and file an issue on source_gen
-if approppriate.""",
+          ''
```

Comment:
I actually put this so the entire rest of the message is not formatted strangely - it's sort of a cludge with dartfmt. I can remove if you like, but then the error message doesn't stick into one paragraph.